connect/startup.sh

```diff
@@ -11,12 +11,13 @@ deactivate() {
 trap deactivate EXIT
 
 # Activate License
+CONNECT_LICENSE_FILE_PATH=${CONNECT_LICENSE_FILE_PATH:-/etc/rstudio-connect/license.lic}
```

Comment:
```suggestion
RSC_LICENSE_FILE_PATH=${RSC_LICENSE_FILE_PATH:-/etc/rstudio-connect/license.lic}
```

The other place we can set this default (the other place that we set it) is in the docker image build itself. I kinda like this approach though